src/org/joml/AABBd.java

```diff
@@ -283,6 +283,55 @@ public double getMin(int component) throws IllegalArgumentException {
         }
     }
 
+    /**
+     * Return the length along the x component
+     *
+     * @return length in the x dimension
+     */
+    public double lengthX(){
+        return maxX - minX;
```

Comment:
this might return a negative value if the AABB is "the wrong way around". As this would also be an invalid AABB I guess this invariant is fine. Or, put a bit differently, the length will always be positive if the AABB is valid.
Not sure whether this invariant should be stated in the docstring.